### PR TITLE
Bump OCP versions to minVersion: 4.14, maxVersion 4.18

### PR DESCRIPTION
**Pull request by mgencur**
OCP 4.13 will be unsupported soon.
We also need 4.18 Catalog for pre-release testing.

### CODE REVIEW

File: templates/csv.yaml [hunk 1]
```diff
@@ -190,7 +190,7 @@ spec:
     - kafka
   links:
     - name: Documentation
-      url: https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html/serverless/index
+      url: https://docs.redhat.com/en/documentation/red_hat_openshift_serverless/__VERSION_MAJOR_MINOR__/html/installing_openshift_serverless/index
```
Comment (mgencur) [inline templates/csv.yaml:193]: We changed the other links in CSV some time ago. This one was forgotten.

**Review by creydr — COMMENTED** [review]
/lgtm

**Comment by mgencur** [comment]
@creydr I had to rebase. Please check again.

**Pull request merged**